src/v/resource_mgmt/logger.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "seastarx.h"
+
+#include <seastar/core/print.hh>
```

Comment:
Why is this header included?

Reply:
I have no idea... `coc.nvim` likes to auto add missing headers. Or at least headers it thinks are missing. In this case is definitely looks like a false positive. Will remove the header.